sdk/samples/iot/generate_certificate.ps1

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+Write-Warning "Certificates created by this script MUST NOT be used for production."
```

Comment:
Also while we're being careful please add a `$errorActionPreference="stop"` (assuming PowerShell core supports which I think it would).  This will make it so that PowerShell native commands like the Get-Content, if they fail, will cause the entire script to stop execution.